pyproject.toml

```diff
@@ -165,6 +165,8 @@ ignore = [
 
 [tool.mypy]
 # See https://mypy.readthedocs.io/en/stable/config_file.html
+# TODO: remove when MyPy fixed (see https://github.com/python/mypy/issues/17166)
+disable_error_code = ["call-arg"]
```

Comment:
It doesn't look like python/mypy#17166 is fully resolved yet - python/mypy#17825